boards/nordic/nrf52840dk/README.md

```diff
@@ -13,21 +13,32 @@ First, follow the [Tock Getting Started guide](../../../doc/Getting_Started.md)
 
 JTAG is the preferred method to program. The development kit has an
 integrated JTAG debugger, you simply need to [install JTAG
-software](../../../doc/Getting_Started.md#optional-requirements).
+software](../../../doc/Getting_Started.md#loading-the-kernel-onto-a-board).
 
 ## Programming the kernel
 Once you have all software installed, you should be able to simply run
 make flash in this directory to install a fresh kernel.
 
 ## Programming user-level applications
-You can program an application via JTAG using `tockloader`:
+You can program an application via over USB using the integrated JTAG using `tockloader`:
 
 ```bash
 $ cd libtock-c/examples/<app>
 $ make
 $ tockloader install --jlink --board nrf52dk
 ```
 
+The same options (`--jlink --board nrf52dk`) must be passed for other tockloader commands
+such as `erase-apps` or `list`.
+
+Viewing console output on the nrf52840dk is slightly different from other boards. You must use
+```bash
+$ tockloader listen --board nrf52dk
```

Comment:
Why is this? I've never had to do this. I can't think of what that would even do in Tockloader.

Reply:
oh wait you're right. You don't actually need to pass the board option. You just need to press the reset button. I'll update, sorry.

Reply:
Why do you have to press the reset button? Just to get the board to start from the beginning again?

Reply:
Yeah. On other boards `tockloader listen` triggers a reset, but not on the nrf52840dk for some reason. I posted an issue in the tockloader repository about this.

Reply:
(This came up because two undergrads working with Phil are attempting to use the nrf52840dk and both emailed me unable to figure out how to get console output to work)

Reply:
Gotcha. I think then it makes sense to clarify that the board will not reset in response to tockloader listen, and if you want output from boot then a manual restart is required.

Reply:
Should I post a separate issue about the process console not working with this board?

Reply:
Does that just need to be added to the main.rs file?

Reply:
...yep. My bad (again)